client/app/scripts/superdesk-packaging/views/add-to-package.html

```diff
@@ -0,0 +1 @@
+<div sd-add-package-dropdown></div>
```

Comment:
We had a decision to use directives as attributes. Unless it's changed, this should stay.